shared/src/libs/packages/http/libs/enums/http-error-message.enum.ts

```diff
@@ -1,9 +1,15 @@
+/* eslint-disable @typescript-eslint/quotes */
 const HttpMessage = {
   USER_EXISTS: 'User already exists',
   BUSINESS_ALREADY_EXISTS: 'Owner already has business!',
   NAME_ALREADY_REGISTERED: 'Business with such name already exists!',
   INVALID_USER_GROUP: 'User of the group cannot create business!',
   BUSINESS_DOES_NOT_EXIST: 'Business does not exist!',
+  UNAUTHORIZED: "You're not authorized",
```

Comment:
Let's use `'You are....'` and enable eslint

Reply:
:D